test_cases/assert_violations/constructor_create_argument.yaml

```diff
@@ -1,8 +1,9 @@
-description: "Assert violation after call to concretely created contract with argument"
+description: Assert violation after call to concretely created contract with argument
 issues:
-- id: "SWC-110"
+- id: SWC-110
   count: 1
   locations:
-  - bytecode_offsets: [277]
-    line_numbers: [11]
-
+  - bytecode_offsets:
+      '0x05ca14f58db1545d83289b13561ace31ca7bee82f9e7a1b0c931d415dcb3ddf7': [277]
```

Comment:
I think this should be the following instead:
295:1:0 (0xa9bafdd71a4744d05c44b4cf2f75e05bb66b0252ae3ec370587e49b15fde453c)